contracts/governance/Governor.sol

```diff
@@ -110,24 +110,34 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor {
      * @dev See {IGovernor-state}.
      */
     function state(uint256 proposalId) public view virtual override returns (ProposalState) {
-        ProposalCore memory proposal = _proposals[proposalId];
+        ProposalCore storage proposal = _proposals[proposalId];
```

Comment:
Note: this is more gas effective in the current format, where proposal is not packed and uses 3 slots. If proposal are packed in an upcoming breaking change, then this should be changed back to memory.